draft-ietf-httpbis-http2bis.xml

```diff
@@ -619,18 +619,40 @@
           which, if blocked by the transmission of a large frame, could affect performance.
         </t>
       </section>
-      <section anchor="HeaderBlock">
-        <name>Field Section Compression and Decompression</name>
+      <section anchor="FieldBlock">
+        <name>Header Compression and Decompression</name>
```

Comment:
I feel like "Field Section Compression and Decompression" would be more consistent.